.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,17 @@
+---
+name: User story
+about: Suggest a feature from the perspective of a specific user role
+title: "[User story]"
+labels: ''
+assignees: ''
+
+---
+
+As a [user], 
+
+I want to [functionality], 
+
+so that [benefit].
```

Comment:
Any reason we should have these on separate lines instead of one?

Reply:
Not really. That was just how it was set up when they showed us at the workshop. One line instead is fine by me.

Reply:
```suggestion
As a [user], I want to [functionality], so that [benefit].
```